### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by zunairkhan811**
The code has been made according to the requirements:
Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.

### CODE REVIEW

File: teacher.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require_relative 'person'
 class Teacher < Person
-  def initialize(specialization)
-    super(id, age, name, parent_permission: true)
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
     @specialization = specialization
   end
```
Comment (ishimwezachee) [inline teacher.rb:6]: - [ ]  Good job to create person, but seemed like there one issue happening, that when I try to add specialization to a teacher. Here is the issue that you need to fix for your app to work properly. 
image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @zunairkhan811  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## Highlights
 ✅ Student are being created successfully 
 ✅ Rentals to a student is created successfully
 ✅ I can list person and books created 
 ✅ Good PR title and description  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: teacher.rb (hunk 1 above)
Comment (ishimwezachee) [reply teacher.rb:6]: Hello @zunairkhan811 , My apology that I mistakenly approved your project in the dashboard and I have reported to my seniors to fix that ,  work on the requested changes for you to proceed . Thank you for your patience.

Comment (zunairkhan811) [reply teacher.rb:6]: Thanks! it has been fixed now

**Review by AbbasSarwar — APPROVED** [review]
Hi @zunairkhan811 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**Pull request merged**